tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+  "include"        : ["./lib"],
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "commonjs",
+    // "lib": [],                             /* Specify library files to be included in the compilation. */,
+    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
+     "outDir": "./dist",
+     "rootDir": "./lib",
+     "removeComments": false,
+    // "noEmit": true,                        /* Do not emit outputs. */
+
+    /* Strict mode */
+    "strict": false,                           /* Enable all strict type-checking options. */
```

Comment:
Strict mode is preferred.

Reply:
Yes, I'll change it once TSC gives no errors for strict false.